src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components"
+
+export const Containerheder = styled.div` 
+position: fixed;
+height: 64px;
+top: 0;
+left: 0;
+z-index: 3;
+display: flex;
+padding: 0 12px;
+justify-content: space-between;
+align-items: center;
+background-color: white;
+box-shadow: 0px 2px 4px #00000026;
```

Comment:
I think the color code has redundant digits (it may be 6)

Reply:
I will check